### PR TITLE
main page clean up

### PR DESCRIPTION
**Pull request by cpauvert**
Hello there, 
Here is an attempt to start cleaning the markdown document #18
Any comments and insights would be appreciated!
BEst regards,

### CODE REVIEW

**Review by magelm — APPROVED** [review]
Looks neat! I approve moving license down and markdown conversion

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # NFDI4Microbiota - Metadata Standards
 
+> [!TIP]
+> Find out about research data management, FAIR principles for microbiology and more on our [Knowledge base](https://knowledgebase.nfdi4microbiota.de)!
```
Comment (magelm) [inline README.md:4]: Typing convention is "Knowledge Base"
Is there a quick fix to check across the whole github if the spelling is uniform?

**Review by magelm — CHANGES_REQUESTED** [review]

**Review by magelm — APPROVED** [review]
approve replacing of html lines with md lines

File: README.md [hunk 2]
```diff
@@ -445,7 +464,9 @@ This figure presents an example of categorizing a known lab-cultured
 sample, along with the corresponding minimal metadata. The
 bidirectionality of the categorization framework is highlighted, as it
 enables the linkage between known, commercially available cultures and
-their original sample environments. <br><br>
+their original sample environments.
+
+
 
 ## <span id="license-ontology">Licenses and ontologies section</span>
```
Comment (magelm) [inline README.md:471]:
```suggestion
#<span id="license-ontology">Licenses and ontologies section</span>
```
1 hashtag as it was done for the other spanning sections

**Review by magelm — CHANGES_REQUESTED** [review]
suggestion for span licenses & ontologies

File: README.md (hunk 1 above)
Comment (cpauvert) [reply README.md:4]: Ok thanks will change this. Yes locally but globally I"m not sure, maybe via the search function at the scale of the repository would already indicate to which extent the term was used.

File: README.md (hunk 2 above)
Comment (cpauvert) [reply README.md:471]: Thanks, but IMO, the span HTML tags should be removed as well and the anchor links (for navigation) should be written in proper Markdown syntax, example below.

```markdown
[Licenses and ontologies section](#license-ontology)
```

File: README.md (hunk 1 above)
Comment (cpauvert) [reply README.md]: done in 5846c8f

**Comment by magelm** [comment]
@cpauvert: Done with review you can merge IMO. Let me know if my "changes requested" status hindering merging. Thanks for the work!

**Pull request merged**